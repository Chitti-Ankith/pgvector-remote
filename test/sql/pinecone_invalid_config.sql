SET enable_seqscan = off;
ALTER SYSTEM RESET pinecone.api_key;
SELECT pg_reload_conf();
CREATE TABLE t (val vector(3));
CREATE INDEX i2 ON t USING pinecone (val) WITH (spec = '{"serverless":{"cloud":"aws","region":"us-west-2"}}');
ALTER SYSTEM SET pinecone.api_key = '5b2c1031-ba58-4acc-a634-9f943d68822c';
SELECT pg_reload_conf();
CREATE INDEX i2 ON t USING pinecone (val);
DROP TABLE t;