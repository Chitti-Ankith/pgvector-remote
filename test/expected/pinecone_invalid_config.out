SET enable_seqscan = off;
ALTER SYSTEM RESET pinecone.api_key;
SELECT pg_reload_conf();
 pg_reload_conf 
----------------
 t
(1 row)

CREATE TABLE t (val vector(3));
CREATE INDEX i2 ON t USING pinecone (val) WITH (spec = '{"serverless":{"cloud":"aws","region":"us-west-2"}}');
ERROR:  Pinecone API key not set
HINT:  Set the pinecone API key using the pinecone.api_key GUC. E.g. ALTER SYSTEM SET pinecone.api_key TO 'your-api-key'
ALTER SYSTEM SET pinecone.api_key = '5b2c1031-ba58-4acc-a634-9f943d68822c';
SELECT pg_reload_conf();
 pg_reload_conf 
----------------
 t
(1 row)

CREATE INDEX i2 ON t USING pinecone (val);
ERROR:  Spec cannot be empty
DROP TABLE t;